app/Filament/Resources/UserResource.php

```diff
@@ -185,6 +186,8 @@ function (Builder $query, array $data) {
             ->bulkActions([
                 BulkActionGroup::make([
                     DeleteBulkAction::make(),
+                    AssignTeamBulkAction::make()
+                        ->visible(fn () => auth()->user()->can('user.*.update', User::class)),
```

Comment:
`can` is supposed to be used to check the result of a Models Policy. Not a permission like this.